xml/System/IntPtr.xml

```diff
@@ -1853,7 +1859,7 @@ For <xref:System.Numerics.IFloatingPoint%601>, this method matches the IEEE 754:
       </ReturnValue>
       <Docs>
         <summary>Gets the size of this instance.</summary>
-        <value>The size of a pointer or handle in this process, measured in bytes. The value of this property is 4 in a 32-bit process, and 8 in a 64-bit process. You can define the process type by setting the <see langword="/platform" /> switch when you compile your code with the C# and Visual Basic compilers.</value>
+        <value>The size of a signed integer in the executing process, measured in bytes. The value of this property is 4 in a 32-bit process, and 8 in a 64-bit process. You can define the process type by setting the <see langword="/platform" /> switch when you compile your code with the C# and Visual Basic compilers.</value>
```

Comment:
```suggestion
        <value>The size of a signed integer in the executing process, measured in bytes. The value of this property is 4 in a 32-bit process, and 8 in a 64-bit process. You can define the process type by setting the <see langword="/platform" /> switch when you compile your application with the C# and Visual Basic compilers.</value>
```
? To make it clearer that class-libraries aren't affected by this.

Reply:
Class libraries are effected by this. You can compile libraries to be `x86` or `x64` specific and to not support `AnyCPU`.